# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1346338387 -3600
#      Thu Aug 30 15:53:07 2012 +0100
# Node ID 6be46886099fa5309b09a0498a6a37e7b74c6299
# Parent  789c8c3c9af50ad83da27551ea6709ba1e45e985
Include oct-base64.h in C++ source file to activate OCTAVE_API decorator.

* liboctave/oct-base64.cc (oct-base64.h): New include.
(Array.h): Use double-quoted inclusion, for consistency with other .cc files.

diff --git a/liboctave/oct-base64.cc b/liboctave/oct-base64.cc
--- a/liboctave/oct-base64.cc
+++ b/liboctave/oct-base64.cc
@@ -23,17 +23,18 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <algorithm>
 
 #include <base64.h>
 
-#include <Array.h>
+#include "Array.h"
+#include "oct-base64.h"
 
 bool
 octave_base64_encode (const char *inc, const size_t inlen, char **out)
 {  
   bool ret = false;
 
   size_t outlen = base64_encode_alloc (inc, inlen, out);
   
